carbonmark-api/src/routes/users/put.ts

```diff
@@ -21,6 +21,19 @@ const handler = (fastify: FastifyInstance) =>
           profileImgUrl && profileImgUrl.length ? profileImgUrl : null,
       };
 
+      // Check if the handle already exists in our database
```

Comment:
we only want to do this for `POST` (new user creation). `PUT` is for edits, this will incorrectly throw for all edits because the handle already exists.

Actually this draws my attention to an existing mistake in this PUT endpoint-- we shouldn't pass `handle` to firestore.update()-- the original requirement was to not support handle changes after profile creation

Reply:
Fair enough, updated and removed handle.